CIScripts/Test/Tests/Integration/DockerDriver.Tests.ps1

```diff
@@ -58,6 +55,9 @@ Describe "Docker Driver" {
         $Session = $Sessions[0]
 
         Initialize-PesterLogger -OutDir $LogDir
+
+        $Modules = Get-ChildItem -Recurse -Filter "*.test.exe"
```

Comment:
1. This should be called on testbed (you need to go deeper) - there are no "*.test.exe" files on tester.
2. Invoke-DockerDriverUnitTest expects "pure" module name (it adds "test.exe" to module name).